src/sass/component_styles/_nav.scss

```diff
@@ -13,12 +13,41 @@ nav {
   }
 }
 
+.navItem {
+  padding: 0 2%; 
+  margin: 0 auto;
+} 
+
+.navItem:hover {
+  color: $color-yellow;
+}
+// Nav typography
+
+.navItem {
+  color: white;
+  font-size: 2.4rem;
+  font-family: $main-type;
+  transition: all 1s;
+
+  @include respond(phone) {
+      color: $color-black; 
+  }
+}
+
 .navAnchor {
   display: flex;
-  padding-left: 1%;
 }
 
-.navItem {
-  padding: 0 2%; 
-  margin: 0 auto;
-} 
\ No newline at end of file
+.navTitle {
+  font-size: 3.2rem;
+  color: $color-yellow; 
+  font-family: $limited-title;
+  letter-spacing: -2.0px;
+  position: relative;
+  left: 12%;
+  padding-top: 2%;
+}
+
+.navTitle-link {
```

Comment:
Learned this handy trick to update the color of a Link's default text-decoration (from purple/blue depending on if it's been visited). 

Which leads me to ask: How can I set up a different color for a visited status? I don't see why I would want to do it here, but I may want to elsewhere.